### DISCUSSION TITLE
How to import and possibly expand on the Account contract?

### DISCUSSION THREAD

**Discussion by lsbyerley** [OpenZeppelin/cairo-contracts#211] [Q&A] [answered]
Hey all.. new to the cairo/starknet ecosystem. Coming from solidity, I've been able to import a contract, expand on it, and then compile the contract. 

```
import "@openzeppelin/contracts/token/ERC721/ERC721.sol";

contract Token is ERC721 {
    // constructor and other added functions
}
```

What would be the cairo equivalent to this? Let's say I want to import the `openzeppelin/account/Account.cairo` contract, expand on it and compile it. I can't seem to find an example of this. Am i thinking of this whole process incorrectly?  The readme says to run `nile compile` .. but that looks into the contract directory. So i guess i assumed i should create a contract in the contracts directory that imports the cairo account contract (assuming i want to compile an account contract).

I have this repo cloned, with a python env and nile setup. Just wondering how to proceed from there

**Comment by martriay** [comment, marked as answer]
Hello! There are two answers to your question:

* The library is still in experimentation phase and the readme warns users to use at their own risk, that's why there is no official release/easy way to install it yet. _expect one really soon_
* Since there's no language-level inheritance/mixin in Cairo, our contracts follow our own [contract extensibility pattern™️](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/Extensibility.md).

**Reply by lsbyerley** [reply]
Thank you for the information @martriay! I'll just use the base account contract for now!
